tests/console/yast2_ftp.pm

```diff
@@ -70,7 +70,8 @@ sub run {
     select_console 'root-console';
 
     # install vsftps
-    zypper_call("in vsftpd yast2-ftp-server", timeout => 180);
+    zypper_call("in yast2-ftp-server vsftpd yast2-users", timeout => 180);
```

Comment:
When yast2-users is needed only on SLE15 I would install it only there.

Reply:
I would do it like here, try to install vsftpd without yast2-users in if condition when it will fail run it as now with record_soft_failure within if
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/caasp/one_line_checks.pm#L48